packages/create-analog/template-angular-v17-blog/src/app/pages/blog/index.page.ts

```diff
@@ -0,0 +1,34 @@
+import { Component } from '@angular/core';
+import { injectContentFiles } from '@analogjs/content';
+import { NgForOf } from '@angular/common';
+import PostAttributes from '../../post-attributes';
+
+@Component({
+  selector: 'app-blog',
+  standalone: true,
+  imports: [NgForOf],
```

Comment:
New Control Flow here?

Reply:
I was thinking about that... are we ok using dev preview features? 🤔

Reply:
Good point. I'm good with using it for a blog template. We can always change the template if its confusing

Reply:
Awesome :) just pushed that change as well as renamed the template directroy to `template-blog`.